build.gradle.kts

```diff
@@ -17,8 +17,18 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-rootProject.group = "com.airthings.lib"
-rootProject.version = "0.2.6"
+val autoVersion = project.property(
+    if (project.hasProperty("AUTO_VERSION")) {
+        "AUTO_VERSION"
+    } else {
+        "LIBRARY_VERSION"
+    },
+) as String
+
+group = "${properties["GROUP"]}"
+version = autoVersion
+
+val iosFrameworkName = "${properties["IOS_FRAMEWORK_NAME"]}"
```

Comment:
Is this only for iOS? What then will be the Android framework name?

Reply:
In settings.properties we have

```
rootProject.name = "kmplog"
```

Reply:
Then maybe we should have it lower-cased for iOS too? Just in case some weird side effects happen due to the case change.

Reply:
I would avoid that because then you do `import kmplog` on iOS and that's not the general practice.